docs/disaster_recovery.md

```diff
@@ -33,3 +33,9 @@ Also, by default, the DR VM will have a "Disaster Recovery" tag.
 If you boot a copy of your production VM, be careful: if they share the same static IP, you'll have troubles.
 
 A good way to avoid this kind of problem is to remove the network interface on the DR VM and check if the export is correctly done.
+
+## HA disabled on replicated VMs
+
+Replicated VMs HA is taken into account by XS/XCP-ng. To avoid the resultant troubles, the property will be removed from the replicated VMs and a tag indicating this change will be added.
```

Comment:
I don't like duplicated code/explanation, can we find a place to put this that could fit for both CR and DR?

Reply:
I will move it to a dedicated section in the backup file.